src/adminDashboard/adminDashboard.html

```diff
@@ -3,8 +3,61 @@
     <div show.bind="!loading">
       <div if.bind="!connected"><button class="button1" click.delegate="connect()">Connect...</button></div>
       <div else class="formContainer">
-        <div if.bind="seeds.length">[Seeds Dropdown, etc...]</div>
-        <div else>You have not created any Seeds</div>
+          <div if.bind="seeds.length">
+            <div>
+              <dropdown item-changed.call="setSeed(index)"
+                selected-item-index.to-view="selectedSeed.metadata.general.projectName"
+                placeholder="Select a Seed...">
+                <div repeat.for="seed of seeds">${seed.metadata.general.projectName}</div>
+              </dropdown>
+            </div>
+            <div if.bind="selectedSeed.address">
+                <div if.bind="selectedSeed.hasEnoughProjectTokens" class="button1" click.delegate="selectedSeed.fundSeed()">
+                  Fund Project
+                </div>
+                <div class="button1" click.delegate="selectedSeed.withdrawFundingTokens()">
+                  Withdraw Funding Tokens
+                </div>
+                <div if.bind="selectedSeed.whitelisted" class="button1" click.delegate="addWhitelist()">
+                  Add Whitelist
+                </div>
+                <div if.bind="!selectedSeed.isPaused" class="button1" click.delegate="selectedSeed.pauseLaunch()">
```

Comment:
Please be sure you are checking for possible reverts for all of these functions, such as here, where maybe one can't pause if the seed is closed.

Reply:
ohk, I'll update

Reply:
done, but have some doubts, regarding the `if.bind()`, I wanted to use multiple conditions to check for `withdraw` and `retrieveProjectTokens`, I have commented the conditions I would use, condition then be simplified though.
Using this conditions, I'm facing the issue, I think it's not able to recognize `||`.

Reply:
done